flink-kubernetes/src/main/java/org/apache/flink/kubernetes/configuration/KubernetesConfigOptions.java

```diff
@@ -60,6 +60,13 @@
 		.withDescription("Service account that is used by jobmanager within kubernetes cluster. " +
 			"The job manager uses this service account when requesting taskmanager pods from the API server.");
 
+	public static final ConfigOption<String> TASK_MANAGER_SERVICE_ACCOUNT =
```

Comment:
You need to run `mvn package -Dgenerate-config-docs -pl flink-docs -am -nsu -DskipTests` to regenerate the configuration docs.